src/types.rs

```diff
@@ -577,16 +577,22 @@ impl VimCompleteItem {
             snippet: snippet.clone(),
         };
 
+        let menu = lspitem
+            .detail
+            .clone()
+            .unwrap_or_default()
+            .replace("\n", " ");
+
         Ok(Self {
-            word,
-            abbr,
+            word: word
+                .replace(&menu.to_string(), "")
```

Comment:
The problem with this are
- We cannot guarantee `word` will contain `menu`
- It's undefined what's the result after removing `menu`.

This is because both `word` and `menu` are sort of free form texts and we cannot make any assumption of their content.

Reply:
I hope I can somehow guess, but I don't know that much.